accounts/abi/bind/bind_extra_test.go

```diff
@@ -0,0 +1,107 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bind_test
+
+import (
+	"bytes"
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/coreth/accounts/abi"
+	"github.com/ava-labs/coreth/accounts/abi/bind"
+	"github.com/ava-labs/coreth/accounts/abi/bind/backends"
+	"github.com/ava-labs/coreth/eth/ethconfig"
+	"github.com/ava-labs/coreth/ethclient/simulated"
+	"github.com/ava-labs/coreth/node"
+	"github.com/ava-labs/coreth/params"
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+// TestGetSenderNativeAssetCall checks that the NativeAssetCall proxies the
+// caller address This behavior is disabled on the network and is only to test
+// previous behavior. Note the test uses [params.TestApricotPhase2Config].
+func TestGetSenderNativeAssetCall(t *testing.T) {
+	// pragma solidity >=0.8.0 <0.9.0;
+	// contract GetSenderNativeAssetCall {
+	// 	address _sender;
+	// 	function getSender() public view returns (address){
+	// 			return _sender;
+	// 	}
+	// 	function setSender() public {
+	// 			_sender = msg.sender;
+	// 	}
+	// }
+	const rawABI = `[
+			{
+					"inputs": [],
```

Comment:
nit: wspace seems off

Reply:
Bundled in https://github.com/ava-labs/coreth/pull/913